scripts/deploy.sh

```diff
@@ -99,6 +99,8 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   then
     rm /etc/openstack_deploy/conf.d/swift.yml
```

Comment:
Ya, that could go here, it's not a problem right now though, so I'd like to keep it as is (since it passed, etc).